app/components/nav-bar.js

```diff
@@ -8,5 +8,21 @@ export default Component.extend({
       this.get('authManager').logout();
       this.get('routing').transitionTo('index');
     }
+  },
+
+  didInsertElement() {
+    this._super.call(this);
+    this.$('.notification.item').popup({
+      popup : '.popup',
+      on    : 'click'
+    });
+  },
+
+  willDestroyElement() {
+    this._super.call(this);
+    this.$('.notification.item').popup({
```

Comment:
@niranjan94 I was using `<a>` tags, for the header to simulate the link functionality which they, will be. I _assumed_ it will mandatorily have to be a link. Semantic UI header links are blue by default. So I was trying to override them. Thank you for the clarification.
Is this much linespacing good to go?
![image](https://cloud.githubusercontent.com/assets/17252805/26273881/10d1a1c8-3d65-11e7-9e56-c8202ceb3f0f.png)